crates/bevy_transform/src/components/transform.rs

```diff
@@ -314,7 +314,7 @@ impl Transform {
     /// If this [`Transform`] has a parent, the `axis` is relative to the rotation of the parent.
     #[inline]
     pub fn rotate_axis(&mut self, axis: Dir3, angle: f32) {
-        self.rotate(Quat::from_axis_angle(axis.into(), angle));
+        self.rotation = ((Quat::from_axis_angle(axis.into(), angle)) * self.rotation).normalize();
```

Comment:
Link the issue here in a comment please; this normalization looks out of place otherwise.

Reply:
Done! Is it fine like this or should I add the full link?